backend/app/invites.py

```diff
@@ -141,6 +141,7 @@ def invite_developer(
 
     worker.send_email.send(
         EmailInfo(
+            message_id=f"{app.app_id}/{invited_user.id}/invited",
```

Comment:
these message id don't look very unique. Uniqueness is paramount here.

Reply:
the invite cases should now be better